backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inout/invoicecandidate/M_InOutLine_Handler.java

```diff
@@ -823,7 +821,7 @@ public static PriceAndTax calculatePriceAndTax(
 				.taxCategoryId(pricingResult.getTaxCategoryId())
 				//
 				.priceEntered(pricingResult.getPriceStd())
-				.priceActual(pricingResult.getPriceStd())
+				.priceActual(inoutLine.getC_OrderLine_ID() != 0 ? inoutLine.getC_OrderLine().getPriceActual() : pricingResult.getPriceStd())
```

Comment:
you clarified with mark the we shall go *not* with the invoice-candidate's price?